Assets/Resources/attackArea.prefab

```diff
@@ -68,6 +68,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 689a34cdfc26dda4ba4f7c983c2aa16b, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  with: enemy
+  damageAmount: 2
```

Comment:
dont change other game objects this card is not associated with, this causes merge conflicts